src/datadog/remote_config/remote_config.cpp

```diff
@@ -173,28 +159,41 @@ std::vector<ConfigMetadata> RemoteConfigurationManager::process_response(
     if (client_configs_it == json.cend()) {
       if (!applied_config_.empty()) {
         std::for_each(applied_config_.cbegin(), applied_config_.cend(),
```

Comment:
configurations should be applied by config key alphabetical order, not by the order them they appear in the response

Reply:
I did not found any mention of the iteration order, can you indicate where is it specified?

Reply:
I can't seem to find it, there are now more than 1000 documents referencing remote config... I didn't dream it, but maybe it applies only to the order for merging configuration (data, rules, etc) for AppSec. Since appsec applies config only at the end, and it merges the data only then, the iteration order _here_ doesn't actually make much of a difference. If I find the reference, I'll leave it here.